components/brave_stats/browser/brave_stats_updater_util.h

```diff
@@ -30,6 +30,16 @@ base::Time GetYMDAsDate(const base::StringPiece& ymd);
 
 std::string GetAPIKey();
 
+enum : uint8_t {
+  IsInactiveUser = 0,
```

Comment:
the names violate the code style https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
fixed